### PR TITLE
fix cli --port issue

### PR DESCRIPTION
**Pull request by AlexMunoz**
fixes https://github.com/slinkity/slinkity/issues/154 

As port is not in eleventyArgs, it has been ignored..

I also defined the default port value directly in the option.

### CODE REVIEW

File: packages/slinkity/cli/index.js [hunk 1]
```diff
@@ -55,7 +55,8 @@ const slinkityArgs = {
   },
   port: {
     flag: '--port <port>',
-    description: 'Port for Vite server (default: 3000)',
+    description: 'Port for Vite server (default: 8080)',
```
Comment (Holben888) [inline packages/slinkity/cli/index.js:58]:
```suggestion
    description: 'Port for Vite server',
```

Adding `(default: 8080)` will actually make the default print _twice_ in the `--help` page! Adding a `defaultValue` below generates this text already. I'd suggest removing the `(default...)` text from here and leaving that `defaultValue`

File: packages/slinkity/cli/index.js [hunk 2]
```diff
@@ -65,9 +66,10 @@ const toEleventyOptions = (allOptions) => {
   for (let [name, value] of Object.entries(allOptions)) {
     if (eleventyArgs[name] != null) {
       eleventyOptions[name] = value
-    }
-    if (name === 'serve' && value === true) {
+    } else if (name === 'serve' && value === true) {
       eleventyOptions.watch = true
+    } else if (name === 'port') {
+      eleventyOptions[name] = value
```
Comment (Holben888) [inline packages/slinkity/cli/index.js:72]: Ah, I see what's going on here. I think I should explain the diff between `slinkityArgs` and `eleventyArgs` briefly:
- `eleventyArgs` are flags passed directly to our eleventy config, unchanged
- `slinkityArgs` are flags that we intercept and process ourselves

There was a time when we managed our own dev server, so we needed to intercept `--port` to be handled separately. This is no longer the case! For a more sustainable solution, I'd suggest moving the `port: {...}` config above _out_ of the `slinkityArgs` object to the `eleventyArgs` object. This avoids the need for a new if statement here.

**Review by Holben888 — COMMENTED** [review]
Left a few suggestions, but this is a great contribution!

**Comment by AlexMunoz** [comment]
Hello @Holben888  i made the changes :+1:

**Review by Holben888 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**